README.md

```diff
@@ -21,10 +21,13 @@ Step 4
 ---
 Make a change to the file and save it then run the following commands to the remote repo:
 ```bash
-git checkout -b mycoolfeature
+git checkout -b mycoolnewfeature
```

Comment:
Looks really cool! remember the '-b' creates a new brach. 
You can always switch branches with 
git checkout <branch-name>